### PR TITLE
complete getAllEndpoints (task3) and implement test to check whether …

### PR DESCRIPTION
**Pull request by brunoFernandes21**
…we get back accurate JSON object

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -30,9 +31,24 @@ describe("GET /api/topics", () => {
...
+
+describe("GET /api/", () => {
+  test("200: responds with an object describing all the available endpoints on the API", () => {
+    return request(app)
+      .get("/api/")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toHaveProperty(["GET /api"], expect.any(Object));
+        expect(body).toHaveProperty(["GET /api/topics"], expect.any(Object));
+        expect(body).toHaveProperty(["GET /api/articles"], expect.any(Object));
```
Comment (Dutton1990) [inline __tests__/app.test.js:50]: You probably don't need these toHaveProperty assertions if you are expecting the whole body to look like endpoints below! I think that expect below is enough 😄

**Review by Dutton1990 — APPROVED** [review]
Nice job Bruno! Don't forget to keep updating the endpoints.json for future tasks 😄 

You can merge and move on!

File: __tests__/app.test.js (hunk 1 above)
Comment (brunoFernandes21) [reply __tests__/app.test.js:50]: Awesome. Should I go back, change them and make another pull request?

Comment (brunoFernandes21) [reply __tests__/app.test.js:50]: Awesome. Thanks!

**Pull request merged**